### DISCUSSION TITLE
Light Sensors?

### DISCUSSION THREAD

**Discussion by Lorccan** [andesse/hue-clip-api.node-red-flows#43] [Q&A] [answered]
I would like to include the light level from motion sensors in automations, but this characteristic is not covered at the moment (or at least it's not as far as I can see).  Is there a straightforward way of doing this?

**Comment by andesse** [comment, marked as answer]
Yes, debug the output from the sse event as full data object. 

When the motion sensor sends it's data, you can use a change node to move the data path of the light level to payload. 

You can have a look in the existing used change nodes how the scheme works. 

If you have further questions just hit me up. 

Andreas

Edit: like this it works for any data you like to extract from the event stream

**Reply by Lorccan** [reply]
Got it!  I modified the motion sensor subflow template to add a new env variable light_threshold.  I then use that in the second switch node to see whether the reported light level is either ≥ or <   I've also then modified the meaning of the debugs on the output.

It's working well. Thanks again for being so fast to reply (and for the great ideas)!
